### ISSUE TITLE
Interactive Window cell output find match isn't noticeable

### ISSUE THREAD

**Issue by rzhao271** [microsoft/vscode#164915]
1. In an interactive window, run the following:
```py
for i in range(10):
    print(i)
```
2. Activate the find widget and change the find settings by clicking on the funnel so that the find widget searches in the cell output as well.
3. Search `3` and keep finding until it highlights the match in the cell output.
4. :bug: The match is barely noticeable.

![Screenshot showing interactive window after step 4 above](https://user-images.githubusercontent.com/7199958/198705065-37a63368-f46f-4ecd-958a-4fb9a3e80b0f.png)

Ref https://github.com/microsoft/vscode/issues/130587
Ref https://github.com/microsoft/vscode/issues/161925

**Comment by rebornix** [comment]
Should be fixed now.
